package.json

```diff
@@ -38,9 +38,13 @@
     "@typescript-eslint/parser": "^4.28.2",
     "babel-jest": "^27.0.6",
     "eslint": "^7.30.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-prettier": "^4.2.1",
```

Comment:
I don't think these eslint plugins and extensions are being used. Can we remove these deps?

Reply:
Will do.